docusaurus/android_versioned_docs/version-5.x.x/04-compose/03-general-customization/01-chat-theme.mdx

```diff
@@ -85,14 +85,12 @@ setContent {
 }
 ```
 
-In the snippet above, we customized the shapes to be different from the default values. We made both the message types rounded, the input field rectangular and changed the avatar to not be circular.
+In the snippet above, we customized the shapes to be different from the default values. We made the message bubbles rounded, the input field rectangular and the avatar a rounded rectangle.
 
 This snippet above will produce the following screen:
 
 ![Custom ChatTheme Component](../../assets/compose_custom_chat_theme_component.png)
 
-You can see how the input field is rectangular, how all messages have rounded corners, regardless of their position in the list or who sent the message. Finally, the avatars are now a squircle, instead of a circle.
```

Comment:
Cutting down on repetition as previously we basically stated this twice, once before the resulting screenshot, and once after.